deployments/anvilprod/environment.py

```diff
@@ -895,7 +894,7 @@ def mkdict(previous_catalog: dict[str, str],
     mksrc('bigquery', 'datarepo-5dd9d83b', 'ANVIL_ccdg_broad_ai_ibd_daly_louis_wes_20240104_ANV5_202409242011'),
     mksrc('bigquery', 'datarepo-f11a2ad0', 'ANVIL_ccdg_broad_ai_ibd_daly_mccauley_wes_20240104_ANV5_202409242021'),
     mksrc('bigquery', 'datarepo-2b74b327', 'ANVIL_ccdg_broad_ai_ibd_daly_mcgovern_niddk_wes_20240104_ANV5_202409242032'), # noqa E501
-    mksrc('bigquery', 'datarepo-b36de71b', 'ANVIL_ccdg_broad_ai_ibd_daly_mcgovern_share_wes_20240104_ANV5_202409261554'), # noqa E501
+    mksrc('bigquery', 'datarepo-b36de71b', 'ANVIL_ccdg_broad_ai_ibd_daly_mcgovern_share_wes_20240104_ANV5_202409261554', pop), # noqa E501
```

Comment:
I think you should pop the previous snapshot for that dataset. It would be confusing to use a new snapshot for popping an old one.